app/containers/message/Message.stories.tsx

```diff
@@ -907,3 +907,141 @@ export const LongNameUser = () => (
 		/>
 	</>
 );
+
+
+const linksWithEmptySpaceAndLineBreak = {
```

Comment:
You changed Markdown, but tested Message.
Can't we just create new Markdown tests?